hw/ip/rom_ctrl/dv/rom_ctrl_base_sim_cfg.hjson

```diff
@@ -57,6 +57,14 @@
       name: rel_path
       value: "hw/ip/{name}_{variant}/dv"
     }
+    {
+      name: default_vcs_cov_cfg_file
+      value: "-cm_hier {proj_root}/hw/dv/tools/vcs/cover.cfg"
+    }
```

Comment:
I think this is the value it had before? So we probably don't need this override?